src/lib/ui/containers/SignupForm/SignupForm.svelte

```diff
@@ -0,0 +1,158 @@
+<script lang="ts">
+  import { createEventDispatcher, getContext } from "svelte";
+  import {
+    Errors,
+    Field,
+    FormError,
+    Select,
+    faivform,
+  } from "@tuentyfaiv/svelte-form";
+  import { getTexts } from "$lib/logic/utils/objects.js";
+  import { fieldsSignup } from "$lib/logic/schemas/index.js";
+
+  import type { Readable } from "svelte/store";
```

Comment:
🚫 **[eslint]** <[import/order](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/order.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`svelte/store` type import should occur before import of `$lib/logic/utils/objects.js`
```suggestion
  import type { Readable } from "svelte/store";
  import { getTexts } from "$lib/logic/utils/objects.js";
  import { fieldsSignup } from "$lib/logic/schemas/index.js";

```